MODEL_HUB.md

```
@@ -0,0 +1,75 @@
+
+app/llm-hub/*                        # text hub + per-model + comparisons (shipped)
+app/models/page.tsx                  # umbrella hub (all 7 categories)
+app/models/[category]/page.tsx       # category hub (image, video, audio, voice, embedding, world)
+app/models/[category]/[slug]/page.tsx# per-model page
+app/models.json/route.ts            # agent-readable JSON
+app/models/opengraph-image.tsx      # social card
+
+components/models-hub/*              # CategoryCard, GenModelCard, GenModelExplorer, GenDecisionMatrix
+```
+
+### Categories (7)
```

markdownlint-cli2 (0.22.1)

[warning] Headings should be surrounded by blank lines
Expected: 1; Actual: 0; Below

(MD022, blanks-around-headings)